packages/admin-app/src/components/ElectionSetupWizard/Steps/JointKeyRetrievedStep.tsx

```diff
@@ -26,14 +27,23 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 export interface JointKeyRetrievedStepProps {
-    onNext?: () => void;
+    onNext: (newSubmitElectionRequest: SubmitElectionRequest) => void;
+    submitElectionRequest: SubmitElectionRequest;
 }
 
 /**
  * Joint Key Select Step for Election Setup
  */
-const JointKeyRetrievedStep: React.FC<JointKeyRetrievedStepProps> = ({ onNext }) => {
+const JointKeyRetrievedStep: React.FC<JointKeyRetrievedStepProps> = ({
+    onNext,
+    submitElectionRequest,
+}) => {
     const classes = useStyles();
+
+    const onButtonClick = () => {
+        onNext(submitElectionRequest);
```

Comment:
Yea, this step as-is is completely useless.  It's just a confirmation dialog that doesn't do anything.  The only reason I made this change in this PR is because onNext now requires a submitElectionRequest and it won't compile without one.  However, if you're ok with the pattern I described earlier then if you don't mind suspending your reservations on this particular comment for the time being, I'm actively working on this step in my current branch to make it far more usable and to follow that pattern.